packages/controller/main.js

```diff
@@ -3082,6 +3082,18 @@ async function processMessage(msg) {
             break;
         }
 
+        // read licenses from iobroker.net
+        case 'updateLicenses': {
+            tools.updateLicenses(objects, msg.message && msg.message.login, msg.message && msg.message.password)
+                .then(licenses =>
```

Comment:
Please add a debug/info logline here, else we do not see what happened when

Reply:
Added